libcxx/test/std/input.output/string.streams/istringstream/istringstream.members/str.allocator_propagation.pass.cpp

```diff
@@ -0,0 +1,165 @@
+//===----------------------------------------------------------------------===//
```

Comment:
IIRC the changes here are not to make us conforming, so all the tests should be either under `libcxx/test/libcxx` or guarded with `#ifdef _LIBCPP_VERSION`.

Reply:
Per https://github.com/llvm/llvm-project/issues/64644#issuecomment-1748957379, it looks like this is in fact necessary for conformance.